x/did/genesis_test.go

```diff
@@ -61,17 +62,16 @@ func (suite *genesisTestSuite) newDIDDocumentWithSeq(did string) (types.DIDDocum
 	verificationMethodID := types.NewVerificationMethodID(did, "key1")
 	es256VerificationMethod := types.NewVerificationMethod(verificationMethodID, types.ES256K_2019, did, pubKey)
 	blsVerificationMethod := types.NewVerificationMethod(verificationMethodID, types.BLS1281G2_2020, did, []byte("dummy BBS+ pub key"))
-	verificationMethods := []*types.VerificationMethod{
-		&es256VerificationMethod,
-		&blsVerificationMethod,
-	}
-	verificationRelationship := types.NewVerificationRelationship(verificationMethods[0].Id)
-	authentications := []types.VerificationRelationship{
-		verificationRelationship,
-	}
-	doc := types.NewDIDDocument(did, types.WithVerificationMethods(verificationMethods), types.WithAuthentications(authentications))
+
+	authentication := types.NewVerification(es256VerificationMethod, ariesdid.Authentication)
```

Comment:
Does we need to change the `types/genesis.go` generated by `v3/genesis.proto`?
It means that change a `DefaultGenesis` and `Validate`.

Reply:
This is already reflected. The newly defined `DIDDocument` is stored in the `GenesisState` and is implemented to validate it.